tests/python_on_whales/components/test_container.py

```diff
@@ -232,6 +232,22 @@ def test_run_with_preserve_fds(podman_client: DockerClient):
     ):
         assert os.read(read_fd, 7) == b"foobar\n"
 
+def test_run_with_timezone(podman_client: DockerClient):
+    output = podman_client.container.run(
+        "ubuntu", ["date", "+'%Z'"], tz="local", remove=True
+    )
+    # Check the timezone in the container matches that in the host.
+    assert output == datetime.now().astimezone().strftime("'%Z'")
+
+def test_create_start_with_timezone(podman_client: DockerClient):
+    with podman_client.container.create(
+        "ubuntu", ["sleep", "infinity"], tz="GMT", stop_timeout=1
+    ) as container:
+        container.start()
+        output = podman_client.execute(container, ["date", "+'%Z'"])
+        # Check the timezone in the container matches what was specified
+        assert output == "'GMT'"
```

Comment:
I take it GMT was chosen as being different to the default in the `ubuntu` image, which seems to be UTC? Maybe worth a comment